aries-site/src/pages/foundation/icons.mdx

```diff
@@ -10,119 +10,140 @@ import {
 
 <Example code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/foundation/icons/IconSizeExample.js">
   <IconSizeExample />
 </Example>
 
-### Button icons
-
-The icon should be centered aligned with any text that is being used next to it. The correct size icon should be paired with similar size text. When using the Icon as a button both the label and icon will be wrapped within that Button.
-
-<Example code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/foundation/icons/IconSizeExample.js">
-  <IconTextExample />
-</Example>
+## Icon color
 
-### Icon plain color
+Standalone icon components use the default color token `icon-default`.\
+Decorative icons may also use `icon-strong` `icon-weak` `icon-disabled` or `icon-primary` to convey emphasis or state.
 
-The default icon color is 'text-strong' for the corresponding light/dark mode. For some icons there are spcific colors in how they are represented, such as HPE, HP that need to respect the built in colors for those SVGs so the color prop can be given 'plain'.
+Non-semantic icons, such as brand or third-party SVGs, should display their native colors. To preserve their original styling, apply the prop `color="plain"`.
 
 <Example code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/foundation/icons/IconPlainExample.js">
   <IconPlainExample />
 </Example>
 
-### Icon height
+## Status icons
 
-In instances where icons are presented alongside text in a row with `align="start"`, it is necessary to add a `height` property to the icon. The value of the height property should match the t-shirt size of the accompanying text. For example, an icon should receive `height="medium"` when presented alongside `medium` (default) text.
+Status icons are reserved for drawing attention to information about system health.
 
-The `height` property on icon is only necessary when a row uses `align="start"`.
+<Example code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/foundation/icons/IconStatusExample.js">
+  <IconStatusExample />
+</Example>
+
+## Aligning with text
+
+When icons appear inline with text, set the `height` to match the text's size prop to ensure visual alignment.
 
-HPE theme icons are sized to match font-sizes, as opposed to line-heights, in order for a more balanced presentation of icons and text. Applying the `height` property to an icon ensures the necessary top and bottom padding is applied to the icon to align with the line-height of the text.
 
 <Example code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/foundation/icons/IconHeightExample.js">
   <IconHeightExample />
```

Comment:
Wondering if this example should be in the format of our do/don't examples?
<img width="741" height="306" alt="Screenshot 2025-11-17 at 4 34 20 PM" src="https://github.com/user-attachments/assets/d517877d-366f-404a-b38c-e4e2b1055e67" />